Documentation/Creating-your-templates.md

```diff
@@ -28,71 +28,57 @@ $ swiftgen templates list
 colors:
   custom:
   bundled:
-   - literals-swift3
    - literals-swift4
    - literals-swift5
-   - swift3
    - swift4
    - swift5
 coredata:
   custom:
   bundled:
-   - swift3
    - swift4
    - swift5
 fonts:
   custom:
   bundled:
-   - swift3
    - swift4
    - swift5
 ib:
   custom:
    - mytemplate
   bundled:
-   - scenes-swift3
    - scenes-swift4
    - scenes-swift5
-   - segues-swift3
    - segues-swift4
    - segues-swift5
 json:
   custom:
   bundled:
-   - inline-swift3
    - inline-swift4
    - inline-swift5
-   - runtime-swift3
    - runtime-swift4
    - runtime-swift5
 plist:
   custom:
   bundled:
-   - inline-swift3
    - inline-swift4
    - inline-swift5
-   - runtime-swift3
    - runtime-swift4
    - runtime-swift5
 strings:
   custom:
    - mycustomtemplate
   bundled:
-   - flat-swift3
    - flat-swift4
    - flat-swift5
-   - structured-swift3
    - structured-swift4
    - structured-swift5
```

Comment:
I'm guessing you only manually removed those entries from the md file, but we now have more templates (e.g. the objc ones) than that and some are missing here, so maybe it would be a good occasion to re-sync that list from the actual output of `swiftgen templates`